### PR TITLE
Create pump-1.8.md

### PR DESCRIPTION
**Pull request by carolwkc**
https://deploy-preview-2526--tyk-docs.netlify.app/docs/nightly/release-notes/
## Related Issue (if applicable)

Add Pump1.8 release notes

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

**Review by mativm02 — APPROVED** [review]
lgtm! ✅

File: tyk-docs/content/release-notes/pump-1.8.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: Pump v1.8
+menu:
+  main:
+    parent: "Release Notes"
+weight: 300
+---
+## 1.8
+Release date: 2023-05-04
+
+### Summary
```
Comment (letzya) [inline tyk-docs/content/release-notes/pump-1.8.md]:
```suggestion
### Major features
```

File: tyk-docs/content/release-notes/pump-1.8.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+We have also added a config option that allow you to decode the raw requests and responses for all pumps so you don't need to worry about processing them in your data pipeline. For demo mode, there is now an option to generate future data for your convenience.
+
+In this release, we are using a new Tyk storage library to connect to Mongo DB. This would allow us to switch to use the official Mongo Driver very easily in the future.
+
+### Notes on MongoDB v3.x compatibility
+
+We have changed the default MongoDB driver from [mgo](https://github.com/go-mgo/mgo) to [mongo-go](https://github.com/mongodb/mongo-go-driver). The new driver supports MongoDB versions greater or equal to v4. If you are using older version of MongoDB v3.x, please [follow this guide to update the driver type](https://github.com/TykTechnologies/tyk-pump#driver-type).
+
```
Comment (dcs3spp) [inline tyk-docs/content/release-notes/pump-1.8.md:25]:
```suggestion
### Changelog

```

Comment (dcs3spp) [inline tyk-docs/content/release-notes/pump-1.8.md:25]: @caroltyk Add Change log section with added and fixed subsections

```suggestion
### Changelog

```

File: tyk-docs/content/release-notes/pump-1.8.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+### Added
```
Comment (dcs3spp) [inline tyk-docs/content/release-notes/pump-1.8.md]: @caroltyk Make added a subsection of Changelog

```suggestion
#### Added
```

File: tyk-docs/content/release-notes/pump-1.8.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+- Added GraphQL SQL Aggregate Pump.
+- Added Resurface Pump - Resurface can provide context-based security analysis for attack and failure triage, root cause, threat and risk identification based on detailed API logs sent from Tyk Pump.
+- Add config option raw_request_decoded and raw_response_decoded for decoding from base64 the raw requests/responses fields before writing to Pump. This is useful if you want to search for specific values in the raw request/response. Both are disabled by default. This setting is not available for Mongo and SQL pumps, since the dashboard will decode the raw request/response.
+- Add the ability to generate future data in demo mode using --demo-future-data flag. 
+- Remove critical CVE go.uuid vulnerability 
+- Use the latest Tyk storage library to connect to Mongo 
+- Hybrid Pump refactoring - we now have better RPC connection control, testability, and documentation 
+
+### Fixed
```
Comment (dcs3spp) [inline tyk-docs/content/release-notes/pump-1.8.md]: @caroltyk Make fixed a subsection of new Changelog section

```suggestion
#### Fixed
```

**Pull request merged**